cases/some_examples_will_break_abi.md

```diff
@@ -0,0 +1,36 @@
+We recommend you use C-API to expose your library to others when you are writing CPP library because the ABI of CPP library is very fragile and easily to break.
```

Comment:
The ABI of C++ libraries is fragile and it is easy (for human programmers) to break it.